### PR TITLE
README.rst: Add yegor256.com #2924

### PR DESCRIPTION
**Pull request by ashwini0529**
I guess the placement is appropriate for the section Awards. Different Awards section is needed as if any other awards adds up.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -191,7 +191,15 @@ You can find a quick demo of coala here:
 .. |asciicast| image:: https://asciinema.org/a/42968.png
    :target: https://asciinema.org/a/42968?autoplay=1
    :width: 100%
+-----
```
Comment (Makman2) [inline README.rst]: what are these for?

File: README.rst [hunk 2]
```diff
@@ -191,7 +191,15 @@ You can find a quick demo of coala here:
...
+
+======
+Awards
+======
```
Comment (Makman2) [inline README.rst]: why you don't use `*****` just below the header like every other title?

File: README.rst (hunk 1 above)
Comment (Makman2) [inline README.rst]: ah nvm

File: README.rst [hunk 3]
```diff
@@ -191,7 +191,15 @@ You can find a quick demo of coala here:
 .. |asciicast| image:: https://asciinema.org/a/42968.png
    :target: https://asciinema.org/a/42968?autoplay=1
    :width: 100%
```
Comment (Makman2) [inline README.rst]: please add a newline here

**Comment by Makman2** [comment]
Please read our [newcomers guide](http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html) :)

**Comment by Adrianzatreanu** [comment]
Could you please work on your commit message? read coala.io/commit , thanks!

**Comment by Makman2** [comment]
@ashwini0529 btw please check up the issue, I think somebody is already working on that one^^

**Comment by ashwini0529** [comment]
@Makman2 I guess this might be good to go. Also, the issue was pending from 6 days, so I decided to take this up. 
@Adrianzatreanu, read the commit messages docs. I hope the updated commit message is good to go?

**Comment by Makman2** [comment]
@ashwini0529 please still sync up on an issue by leaving a comment there, so we can coordinate that better.

\+ about the commit message: Please put a `Closes <full-link-to-issue>` into the commit body, so GitHub auto closes the issue on merge.

**Comment by ashwini0529** [comment]
@Makman2 , Added `Closes <full-link-to-issue>` in commit body. 
Is there anything else apart from this that I need to do?

**Comment by Nosferatul** [comment]
@ashwini0529 Your rebase went wrong. Take a look https://docs.coala.io/en/latest/Developers/Git_Basics.html or on gitter use `cobot explain rebase`

**Comment by ashwini0529** [comment]
@Nosferatul I guess everything's fine now?

**Comment by ashwini0529** [comment]
@Makman2 Is there anything else that needs to be done?

**Comment by sils** [comment]
can you use the full issue URL in the message, not ust the number please?

**Comment by sils** [comment]
@ashwini0529 @Makman2 @Nosferatul do we really want an own awards section just for this one thing? Did we receive any other awards? I'm not fully bought on this yet.

**Comment by Nosferatul** [comment]
I like how it looks. We should be proud of this 😄 . No but I'm positive we will have more in the future 👍

**Comment by ashwini0529** [comment]
@sils I think we can have a section for awards section. That would be really helpful in future. 
And yeah,   I'll just add the full url of the issue.

**Comment by Makman2** [comment]
btw why am I the main author of the commit? :O

**Comment by ashwini0529** [comment]
@sils @Makman2 please merge the pull request if everything is fine.

**Comment by Adrianzatreanu** [comment]
ack fb7ed29859081d7c1c16413ea5ea14d2bd3d3500

**Comment by Adrianzatreanu** [comment]
hey congratz you just got your first newcomer issue merged. Please take the time and do the following 2 things:
 - fill in coala.io/newform . it helps us a lot and takes a few mins at max :) 
 - keep working on a coala.io/low issue, they are truly rewarding :) if you need help picking one up, let us know on gitter

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by ashwini0529** [comment]
@Adrianzatreanu, filled the form. 
And thanks for merging my pull request. I'm really sorry if I made mistakes throughout the issue. I'm a new comer, will get used to the working style.

**Pull request merged**